tests/helper/helper_filesystem.go

```diff
@@ -108,3 +108,37 @@ func copyDir(src string, dst string, info os.FileInfo) error {
 	_, err = io.Copy(dFile, sFile)
 	return err
 }
+
+// CreateFileAtPathWithContent creates a file at the given path and writes the given content
+// path is the path to the required file
+// fileContent is the content to be written to the given file
+func CreateFileAtPathWithContent(path string, fileContent string) error {
+	// check if file exists
+	var _, err = os.Stat(path)
+
+	var file *os.File
+
+	// create file if not exists
+	if os.IsNotExist(err) {
```

Comment:
You don't need this check.
You can just use `os.OpenFile(path, os.O_RDWR|os.O_CREATE, 0644)`. This will openfile with read&write permission and also create it if it doesn't exist.